emails/models.py

```diff
@@ -318,6 +318,7 @@ class DomainAddress(models.Model):
     address = models.CharField(max_length=64)
     enabled = models.BooleanField(default=True)
     description = models.CharField(max_length=64, blank=True)
+    domain = models.PositiveSmallIntegerField(choices=DOMAIN_CHOICES, default=2)
```

Comment:
Should we change `RelayAddress.domain` to `default=2` also?

Reply:
No, because that will make everyone irregardless of them being a premium user create alias with the mozmail.com. We should do this if/when Tony is ready to have everyone generate the alias with the new domain.